plugin.xml

```diff
@@ -36,7 +36,6 @@
     <header-file src="src/ios/ScannerViewController.h"/>
     <source-file src="src/ios/ScannerViewController.xib"/>
 
-    <framework src="ZXingObjC" type="podspec" spec="~> 3.6.4"/>
+    <framework src="https://github.com/OutSystems/ZXingObjC.git" type="podspec" spec="~> 3.6.4"/>
```

Comment:
In this case, is it not necessary to create an outsystems branch with a 3.6.4-OS release?

Reply:
That is actually a good question. I don't really know. But I think its by the TAG.

Reply:
This changed @CarlsCorrea I can confirm that the Tag works.